ocfweb/main/templatetags/staff_hours.py

```diff
@@ -1,8 +1,20 @@
 from django import template
 
+from ocfweb.templatetags.lab_hours import lab_hours_holiday as holiday_hours
```

Comment:
why renaming this function during import? it's better to avoid this except when necessary as it leads to confusion about what function it really is